versions/1/changelog.yml

```diff
@@ -2,7 +2,7 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom.
 ##
-- version: 1.9.1-next
+- version: 1.10
   changes:
     - description: Prepare for next version
```

Comment:
Btw, I think that these entries to prepare for next versions can be removed.

Reply:
Fixed